lifecycle-operator/controllers/common/helperfunctions.go

```diff
@@ -105,3 +106,11 @@ func getObject(k8sclient client.Client, log logr.Logger, ctx context.Context, de
 	}
 	return nil
 }
+
+func GetControllerInfo(req ctrl.Request) map[string]string {
```

Comment:
I think this could be renamed to better reflect that this returns info about the request handled by a controller, not the controller itself. Also, can you please add a go doc comment (see https://tip.golang.org/doc/comment for more info) that describes the purpose of this function? 
```suggestion
func GetRequestInfo(req ctrl.Request) map[string]string {
```